src/Native/build-native.sh

```diff
@@ -150,8 +150,8 @@ __UnprocessedBuildArgs=
 __CrossBuild=0
 __ServerGC=0
 __VerboseBuild=false
-__ClangMajorVersion=3
-__ClangMinorVersion=5
+__ClangMajorVersion=0
```

Comment:
Hm, what is the difference between setting the default version here and at the other place? If we set it here, it can be overriden by the command line option. If there is no clang option, it would take this default version. 
If we do it this new way, then the effect is the same. If the command line option is used, the version is set accordingly. If it is not set, we set it later to the default one.